Classes/Domain/Manager/AbstractManager.php

```diff
@@ -132,6 +132,7 @@ public function persist(): void
             $this->dataHandler->process_datamap();
             if (count($this->dataHandler->errorLog) !== 0) {
                 $this->logger->error('Error in importing', $this->dataHandler->errorLog);
```

Comment:
I think we can remove the logging part here. The exception is catched by the global ErrorHandler anyways.

Reply:
done